src/main/java/com/google/cloud/tools/gradle/appengine/core/ManagedSdkTask.java

```diff
@@ -0,0 +1,53 @@
+package com.google.cloud.tools.gradle.appengine.core;
```

Comment:
In IntelliJ, in `Preferences -> Editor -> File and Code Templates`, for the `Class`, `Interface`, and `Enum` files, you could insert the copyright notice to the top of the template so that it is automatically added to each new file.